remote-workstations/new-agent-vm/Install-PCoIPAgent.sh

```diff
@@ -243,131 +238,326 @@ active_d = ldap_lib(args.address, args.user_name, args.password, args.domain)
 active_d.add_computer_to_group(args.computer, args.group)
 
 EOF
-sudo python $file_path -d "$DOMAIN_NAME" -a "$DC_ADDRESS" -u "$USERNAME" -p "$PASSWORD" -c "$VM_NAME" -g "$GROUP"
 
+    sudo python $file_path -d "$DOMAIN_NAME" -a "$DC_ADDRESS" -u "$USERNAME" -p "$PASSWORD" -c "$VM_NAME" -g "$GROUP"
+}
+
+install_gui() 
+{
+    # Make sure Linux OS is up to date
+    echo "--> Updating Linux OS to latest"
+
+    sudo yum -y update  # --exclude=WALinuxAgent
```

Comment:
Does this work? We needed to exclude WALinuxAgent before

Reply:
it works

Reply:
it is possible.  Do we need to exclude forever to solve the issue?